frontend/src/index.css

```
@@ -0,0 +1,144 @@
+  --radius-lg: 20px;
+  --radius-xl: 28px;
+
+  --shadow-glow-cyan: 0 0 40px rgba(0,212,255,0.15);
+  --shadow-glow-green: 0 0 40px rgba(0,255,136,0.15);
+  --transition: all 0.25s cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+*, *::before, *::after { box-sizing: border-box; margin: 0; padding: 0; }
+
+body {
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)